homeassistant/helpers/config_validation.py

```diff
@@ -255,6 +256,17 @@ def time_zone(value):
 weekdays = vol.All(ensure_list, [vol.In(WEEKDAYS)])
 
 
+def url(value: Any) -> str:
```

Comment:
Let's not call it url, as a url can contain any protocol. Let's call it `http_url`